live/app/components/course-list.js

```diff
@@ -22,6 +22,7 @@ const CourseList = Ember.Component.extend({
 
   courses: null,
   selectedCourse: null,
+  classNames: ['course-list'],
 
   didInsertElement () {
     const that = this;
```

Comment:
y en a besoin car le `that` est utilisé dans une callback de callback qui ne sont pas des arrow functions.